backend/Controllers/crudController.js

```diff
@@ -113,28 +111,27 @@ exports.joinGroup = async (req, res) => {
   )
     .then((data) => {
       //if updateOne modified
-      if (data.n != 0 ) {
+      if (data.n != 0) {
         //add groupCode to user object
-      User.updateOne(
-        { userID: req.body.spotifyID }, //filter
-        { $addToSet: { groups: req.body.groupCode } }
-      )
-        .then(() => {
-          res.json({
-            message: "Successfully joined group",
-            groupCode: req.body.groupCode,
-          });
-        })
-        //error with adding group to user object
-        .catch((err) => {
-          res.status(400).json({
-            message: "Unable to add group to user object",
-            error: err,
+        User.updateOne(
+          { userID: req.body.spotifyID }, //filter
+          { $addToSet: { groups: req.body.groupCode } }
+        )
+          .then(() => {
+            res.json({
+              message: "Successfully joined group",
+              groupCode: req.body.groupCode,
+            });
+          })
+          //error with adding group to user object
+          .catch((err) => {
+            res.status(400).json({
+              message: "Unable to add group to user object",
+              error: err,
+            });
           });
-        });
-      }
-      else {
-        throw new Error("Group does not exist.")
+      } else {
+        throw new Error();
```

Comment:
Is this the format we should use for future error handling?

Reply:
Yea in our use cases the string didn't actually do anything so I got rid of it. keep in mind this is definitely not bEsT pRaCtIcE lmao